src/lib_json/json_writer.cpp

```diff
@@ -309,31 +309,26 @@ static String valueToQuotedStringN(const char* value, unsigned length,
     // Should add a flag to allow this compatibility mode and prevent this
     // sequence from occurring.
     default: {
-      if (emitUTF8) {
-        result += *c;
-      } else {
-        unsigned int codepoint = utf8ToCodepoint(c, end);
-        const unsigned int FIRST_NON_CONTROL_CODEPOINT = 0x20;
-        const unsigned int LAST_NON_CONTROL_CODEPOINT = 0x7F;
-        const unsigned int FIRST_SURROGATE_PAIR_CODEPOINT = 0x10000;
```

Comment:
It is better to keep so that people know what they mean.

Reply:
I've removed them based on @BillyDonahue suggestion. Now I reverted them back, but please lets no go back and forth all the time.

Reply:
I don't think these need names, particularly names with that style. The 0x1000 is really just a bit pattern. It would be like naming a variable ONE_HUNDRED or something. Frankly I'd rather use comments than variable names if there's risk of confusion for readers. These are locals and make no difference beyond the body of this function.

Reply:
For me, I just need to know what they mean. So it is ok to use comments.
I find you create a new PR for this. Both are great.